contracts/Oracle/Calculations/Sushiswap.sol

```diff
@@ -112,18 +108,12 @@ contract CalculationsSushiswap {
         uint256 amountIn = 10**uint256(token0.decimals());
         uint256[] memory amountsOut;
 
-        bool fallbackRouterExists = secondaryRouterAddress != zeroAddress;
-        if (fallbackRouterExists) {
-            try primaryRouter.getAmountsOut(amountIn, path) returns (
-                uint256[] memory _amountsOut
-            ) {
-                amountsOut = _amountsOut;
-            } catch {
-                amountsOut = secondaryRouter.getAmountsOut(amountIn, path);
-            }
-        } else {
-            amountsOut = primaryRouter.getAmountsOut(amountIn, path);
+        PriceRouter router = defaultRouter;
+        address routerOverrideAddress = routerOverrides[token0Address];
+        if (routerOverrideAddress != zeroAddress) {
+            router = PriceRouter(routerOverrideAddress);
         }
+        amountsOut = router.getAmountsOut(amountIn, path);
 
         // Return raw price (without fees)
         uint256 amountOut = amountsOut[amountsOut.length - 1];
```

Comment:
do we need to worry about the fee logic being different on the line below for uniswap v3 positions? Or should we just assume it's always 0.3%